examples/sparse_finch.ipynb

```diff
@@ -0,0 +1,355 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Finch backend for `sparse`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# pip install 'sparse[finch]==0.16.0a9' scipy\n",
+    "# export SPARSE_BACKEND=Finch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import importlib\n",
+    "import os\n",
+    "import time\n",
+    "\n",
+    "import sparse\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import numpy as np\n",
+    "import scipy.sparse as sps\n",
+    "import scipy.sparse.linalg as splin"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# let's make sure we're using Finch backend\n",
+    "os.environ[sparse._ENV_VAR_NAME] = \"Finch\"\n",
+    "importlib.reload(sparse);"
```

Comment:
Can we move this to before the first `sparse` import and avoid the `importlib.reload` call?

Reply:
Perhaps we should print out the backend here instead then?

Reply:
Done! I added a `print` and `assert` statements.